packages/patternfly-4/_repos/react-core/src/demos/PageLayout/examples/PageLayoutDefaultNav.js

```diff
@@ -192,6 +192,7 @@ class PageLayoutDefaultNav extends React.Component {
           isManagedSidebar
           skipToContent={PageSkipToContent}
           breadcrumb={PageBreadcrumb}
+          className="pf4-site-demo-typography"
```

Comment:
Anything under _repos will get overwritten when newer patternfly-react and core versions are pulled in. So this would only be a temporary solution. Perhaps we can add a class identifier upstream?

Reply:
@jschuler this is really only necessary until PatternFly moves over to using the Red Hat typography (which will then match the rest of PatternFly.org). I don't know the timeline for that (I couldn't find an issue in the patternfly/patternfly-next repo), but it probably wouldn't make sense to put an identifier upstream.

Reply:
@mindreeper2420 there is a PR up in pf next currently that adds an option to use the new font family: https://github.com/patternfly/patternfly-next/pull/1813 .. I dont know how long until it will be merged but I can check with Michael Coker